### PR TITLE
[data factory] add tumblingWindowTrigger dependsOn and add Folder to pipeline and da…

### PR DESCRIPTION
**Pull request by haoyingli**
…taset

This SDK includes 2 swagger changes:
1.  #3636 [Datafactory] Add folders to Pipeline and Dataset
https://github.com/Azure/azure-rest-api-specs/pull/3636
2.  #3445 [Data Factory] add tumbling window dependsOn property under stable
https://github.com/Azure/azure-rest-api-specs/pull/3445

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by hvermis — APPROVED** [review]
:shipit:

**Review by hvermis — APPROVED** [review]
We will need to update the version of SDK separately

File: src/SDKs/DataFactory/DataFactory.Tests/JsonSamples/TriggerJsonSamples.cs [hunk 1]
```diff
@@ -149,6 +149,60 @@ public class TriggerJsonSamples : JsonSampleCollection<TriggerJsonSamples>
     }
   }
 }
+";
+
+        [JsonSample]
```
Comment (dsgouda) [inline src/SDKs/DataFactory/DataFactory.Tests/JsonSamples/TriggerJsonSamples.cs]: nit: please fix indentation

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please update `PackageReleaseNotes` in the csproj

**Comment by dsgouda** [comment]
@haoyingli Please fix failing tests too

File: src/SDKs/DataFactory/DataFactory.Tests/JsonSamples/TriggerJsonSamples.cs (hunk 1 above)
Comment (haoyingli) [reply src/SDKs/DataFactory/DataFactory.Tests/JsonSamples/TriggerJsonSamples.cs]: The current Json samples also use these indentation, 8 spaces before  [JsonSample], what do you want me to change to?

**Review by dsgouda — APPROVED** [review]

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please update `PackageReleaseNotes`

**Review by dsgouda — APPROVED** [review]

**Comment by haoyingli** [comment]
When will this PR be merged? I see the status is "Merging is blocked", is there anything blocking it?

**Pull request merged**